doozer/doozerlib/cli/scan_sources.py

```diff
@@ -307,7 +458,9 @@ def generate_report(self):
         if self.as_yaml:
             click.echo('---')
             click.echo(yaml.safe_dump(results, indent=4))
+            click.echo(yaml.safe_dump({'issues': self.issues}, indent=4))
```

Comment:
This will dump multiple yaml documents into stdout. Not sure if it will create a subtle issue. This may need more testing, or we can combine `issues` to the previous yaml document.

Reply:
Done with e2b238c9c01bf1a95d40be9c5fec5090e61af359